l10n_br_account/models/fiscal_document.py

```diff
@@ -43,6 +43,8 @@ class FiscalDocument(models.Model):
         copy=False,
     )
 
+    incoterm_id = fields.Many2one(related="move_ids.invoice_incoterm_id")
+
```

Comment:
OK, mas então por favor botar um comentario em cima tipo;
`# for some reason (issue with _inherits?) it seems fields.Many2one(related="invoice_incoterm_id") stopped working`

Reply:
vou arrumar :blush: